ui/src/features/AssetsDialog/hooks.ts

```diff
@@ -27,6 +28,9 @@ export default ({
     deleteAsset,
     isCreatingAsset,
   } = useAsset();
+  const availableExtensions = ALLOWED_ASSET_IMPORT_EXTENSIONS.split(",").map(
+    (ext) => ext.trim().replace(/^\./, ""),
+  );
```

Comment:
Just use ALLOWED_ASSET_IMPORT_EXTENSIONS directly. It seems you remove the period, but then everywhere you use availableExtensions, you reapply it.